rust/libnewsboat/Cargo.toml

```diff
@@ -11,6 +11,7 @@ regex = "1.0"
 url = "1.7"
 dirs = "1.0.4"
 backtrace = "= 0.3"
+unicode-width = "0.1.5"
```

Comment:
Please commit an updated Cargo.lock, too. We added it to the repo just recently (#392); it ensures that we all use the same libraries when building Newsboat.

Reply:
Thank you. I didn't know that. I'll push it up.